gnucash/gnome-utils/gnc-dense-cal.h

```diff
@@ -57,6 +57,9 @@ struct _GncDenseCal
 
     gboolean showPopup;
     GtkWindow *transPopup;
+    gint screen_width;
+    gint screen_height;
+    gint doc;
```

Comment:
I'm not very familiar at all with the gnucash codebase, but whilst reading through this pull request I can't figure out what `doc` means or stands for.

I guess my first question is what does `doc` stand for or refer to? And my second question would be if it has a "long-form" name would it make sense to use that instead?